internal/pkg/service/common/etcdop/prefix.go

```diff
@@ -102,6 +103,30 @@ func (v Prefix) DeleteAll(opts ...etcd.OpOption) op.CountOp {
 	)
 }
 
+func (v PrefixT[T]) GetOne(opts ...etcd.OpOption) op.ForType[*op.KeyValueT[T]] {
+	return op.NewGetOneTOp(
+		func(_ context.Context) (etcd.Op, error) {
```

Comment:
`GetOne` nad `Prefix`, ... potrebujem to pre ziskanie posledneho mappingu.
Sort sa nastavuje externe cez `opts`, tu je len `Limit`.